hadoop-hdds/docs/content/OzoneFS.md

```diff
@@ -122,4 +122,7 @@ Hadoop version | Required jar            | OzoneFileSystem implementation
 3.2            | filesystem-lib-current  | org.apache.hadoop.fs.ozone.OzoneFileSystem
 3.1            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.OzoneFileSystem
 2.9            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem
-2.7            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem
\ No newline at end of file
+2.7            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem
```

Comment:
@ajayydv you probably didn't mean to leave the change markers in place here.

Reply:
thanks for catching this, removed!!